dan_layer/storage_sqlite/migrations/2022-10-03-178235_payload/up.sql

```diff
@@ -0,0 +1,96 @@
+create table payload_table {
+    id integer not null primary key AUTOINCREMENT
+    payload_id blob not null,
+    payload blob not null
+}
+
+-- fetching by the payload_id will be a very common operation
+create index payload_index on payload (payload_id);
```

Comment:
Include the names in the index, because there may be multiple indices on a table
```suggestion
create index payload_index_payload_id on payload (payload_id);
```

Reply:
Oh I see, cool. Going to refactor !